ember-style-modifier/src/template-registry.ts

```diff
@@ -1,4 +1,4 @@
-import type StyleModifier from './modifiers/style';
+import StyleModifier from './modifiers/style.ts';
```

Comment:
@jelhan i'm not 100% sure, but i have seen that this files will be exported always empty in other addons...

The glint docs says, that the registry should be added directly in helper / component or?
https://typed-ember.gitbook.io/glint/environments/ember/template-registry#helpers-and-modifiers

Reply:
This is the relevant docs for authoring addons: https://typed-ember.gitbook.io/glint/environments/ember/authoring-addons

We should provide a Template Registry, which consumers can import to register the helper. This is explained here: https://typed-ember.gitbook.io/glint/environments/ember/using-addons#using-glint-enabled-addons

Unless there is some magic in the v2 addon blueprint to generate this file automatically, it should be still needed.

I'm only confused that the import changed from type-only to regular import. That I don't get.

Reply:
I just noticed that the template registry is not consumed in `test-app/types/global.d.ts`. I'm confused. How does the `{{style}}` modifier gets registered in test-app so that glint knows the types for it?

Reply:
i have now changes the file like there was before... because this is correct.. in additional i have `addon.publicEntrypoints(['index.js', '**/*.js']),` with `addon.publicEntrypoints(['index.js', 'modifiers/**/*.js'])`, because `'**/*.js'` has created the empty `template-registry.js`. The used `template-registry.js` for glint is `declarations/template-registry.js` and this was exported in both ways correctly... but we should this what the docs says.

In test-app there was missing a part of `glint`... there was only ts check active in package.json... now everything is correctly in `test-app`

So i think, we have resolved this one